Godeps/Godeps.json

```diff
@@ -609,6 +609,51 @@
 		{
 			"ImportPath": "gopkg.in/yaml.v2",
 			"Rev": "7ad95dd0798a40da1ccdff6dff35fd177b5edf40"
+		},
+		{
+			"ImportPath": "github.com/letsencrypt/challtestsrv/vendor/github.com/miekg/dns",
```

Comment:
This seems like a weird problem with Godep - in theory the challtestsrv dep should be satisfied with the `miekg/dns` that we already have in Godeps / vendor at the top level.

Reply:
I'd rather not update the `challtestsrv` than deal with Godep tbh. It isn't important to this PR.

Reply:
Reverted the dependency update in 8bae756

Reply:
Missed some bits, caught them in 3a3385b